src/fides/api/api/v1/endpoints/messaging_endpoints.py

```diff
@@ -529,26 +529,27 @@ def update_messaging_templates(
     failed = []
 
     for template in templates:
-        key = template.key
+        template_type = template.type
         content = template.content
 
         try:
-            default_template = DEFAULT_MESSAGING_TEMPLATES.get(key)
+            default_template = DEFAULT_MESSAGING_TEMPLATES.get(template_type)
             if not default_template:
-                raise ValueError("Invalid template key.")
+                raise ValueError("Invalid template type.")
 
             content["subject"] = (
                 content["subject"] or default_template["content"]["subject"]
             )
             content["body"] = content["body"] or default_template["content"]["body"]
 
-            MessagingTemplate.create_or_update(
-                db, data={"key": key, "content": content}
+            create_or_update_basic_templates(
+                db,
+                data={"type": template_type, "content": content, "is_enabled": False},
```

Comment:
If you have existing enabled templates this will update them to be disabled -

Reply:
I wanted to ensure that if the Basic template UI is being used, then we're not risking the customer enabling the ENV var, and inadvertently sending out messaging templates that were intended only for Basic template mode. I don't HAVE to do this, but it will align with our migration to use "is_enabled=False" across the board. This way, a customer has to explicitly pick / choose which templates they wish to use for property-specific messaging.

Reply:
okay thanks for explaining your thought process, can you add this as a code comment, so it's clear that disabling every template that gets saved here is intentional?  Especially the bit from the migration comment where you said the "disabled" field is ignored in certain cases.